istio.mixer.adapter.model.v1beta/grpc_adapter_service.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+// Template agnostic service interface.
+package istio.mixer.adapter.model.v1beta;
+
+
+message SetAdapterConfigReq {
+    map<string, google.protobuf.Value> allHandlerConfigsByName = 1;
+}
+
+message ValidateReq {
```

Comment:
Use full spelling Request not Req.

Reply:
Thanks Hong, this is a demo PR. I will update the eventual types to conform to the style guide.